components/mediaquery/overview.md

```diff
@@ -8,25 +8,16 @@ published: True
 position: 0
 ---
 
-# Blazor MediaQuery Overview
+# MediaQuery Overview
 
 The <a href = "https://www.telerik.com/blazor-ui/mediaquery" target="_blank">MediaQuery component for Blazor</a> allows you to react when the user changes the size of the browser. 
 
-#### This article is separated in the following sections:
-
-* [Basics](#basics)
-* [Notes](#notes)
-* [Features](#features)
-* [Examples](#examples)
-
-## Basics
+## Creating MediaQuery
```

Comment:
An important question that we should address from the start is how to configure support for multiple different media queries. So the basic example may include two component instances.